### ISSUE TITLE
Extend your bug report template with entry for cwa version

### ISSUE THREAD

**Issue by ndegendogo** [corona-warn-app/cwa-app-ios#1632]
## Avoid duplicates

* [ ] This issue has not already been raised before
* [ ] If you are proposing a new feature, please do so in [CWA-Wishlist](https://github.com/corona-warn-app/cwa-wishlist)

## Current Implementation

Your current bug reporting template is well structured and has also entries for iOS version and device type of the user. Only an entry for cwa version is missing in that template. So many users forget to provide this very useful information.

## Suggested Enhancement

Add entry for cwa version in the section 'Technical details'

## Expected Benefits

Users reporting a bug will hopefully not forget to provide this information.

**Comment by dsarkar** [comment]
@ndegendogo, good point, indeed. We will raise that issue to the developers in order to adapt the template. Best regards, DS

---
Corona-Warn-App Open Source Team

**Comment by ndegendogo** [comment]
I think also the Android template could be extended with such an entry.
